linux_os/guide/services/ssh/ssh_server/sshd_disable_empty_passwords/rule.yml

```diff
@@ -34,12 +35,14 @@ references:
     disa: CCI-000366,CCI-000766
     hipaa: 164.308(a)(4)(i),164.308(b)(1),164.308(b)(3),164.310(b),164.312(e)(1),164.312(e)(2)(ii)
     nist: AC-17(a),CM-7(a),CM-7(b),CM-6(a)
+    nist@sle15: CM-6 b,CM-6.1 (iv)
```

Comment:
we might have to check if spaces in the references section can be a problem. I've seen that a lot of them proposed in this PR contain spaces.

Reply:
It seems that the build system is able to work with spaces. It splits using comma as the separator (https://github.com/ComplianceAsCode/content/blob/master/ssg/build_yaml.py#L1261).

So, I don't have any strong opinions against it. It's just for the sake of consistency, and I do understand that in the original NIST references the spaces are present. To be honest, I don't know if there was a reason to strip these whitespaces, it goes beyond my time in this project.

Reply:
@ggbecker The spaces should be gone now.